hikari/commands.py

```diff
@@ -249,12 +249,9 @@ class PartialCommand(snowflakes.Unique):
     version: snowflakes.Snowflake = attr.field(eq=False, hash=False, repr=True)
     """Auto-incrementing version identifier updated during substantial record changes."""
 
-    name_localizations: typing.Mapping[typing.Union[locales.Locale, str], str] = attr.field(factory=dict)
+    name_localizations: typing.Mapping[typing.Union[locales.Locale, str], str] = attr.field()
```

Comment:
A bit out of scope of this pr, but maybe worth fixing it too, but these should probably also get `attr.field(eq=False, hash=False, repr=False)`.